itests/framework/framework.go

```diff
@@ -715,7 +716,12 @@ func (f *TestFramework) ExtractFileFromCAR(ctx context.Context, t *testing.T, fi
 		require.NoError(t, err)
 	}
 
-	nd, err := ipld.Decode(b)
+	reg := ipld.Registry{}
+	reg.Register(cid.DagProtobuf, dag.DecodeProtobufBlock)
+	reg.Register(cid.DagCBOR, ipldcbor.DecodeBlock)
+	reg.Register(cid.Raw, dag.DecodeRawBlock)
+
+	nd, err := reg.Decode(b)
```

Comment:
These are by far the most common codecs registered in the go-ipld-format global map which is being removed. Given that you're only doing UnixFS here we could drop the dag-cbor code too and things should be fine.